packages/webcomponents/src/components/payment-details/payment-details.tsx

```diff
@@ -60,17 +63,21 @@ export class PaymentDetails {
     }
   }
 
-  paymentService = new PaymentService();
+  handleError = (event) => {
+    if (event.detail === API_ERRORS.NOT_AUTHENTICATED) {
+      this.tokenExpired.emit({ bubbles: true, composed: true });
```

Comment:
I think we ought to emit something in this event - right now its set up to emit an object with the properties `bubbles: true, composed: true`

Perhaps it should simply emit the event, or the `event.detail`. 

Also, I think you would then move this information to line 42 - so that the event declaration reads as:
`@Event({ bubbles: true, composed: true }) tokenExpired: EventEmitter<any>`

Reply:
You're right. I added this to test something and forgot to remove. It's not necessary. 👍

Reply:
Cool cool - I saw your update. I do still wonder if we will eventually want to emit the `event.detail` or the `error` inside `tokenExpired.emit()` but I guess we can decide that whenever we want. 

Good work, thanks for the update! 

I'll approve for now, but still need to do Dev QA for this one